qiskit_addon_utils/exp_vals/observable_mappings.py

```diff
@@ -0,0 +1,165 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Utility functions for mapping between observables qubit indexing definitions."""
+
+from qiskit.quantum_info import Pauli, SparsePauliOp, SparseObservable
+
+
+def map_observable_isa_to_canonical(isa_observable, canonical_qubits):
+    """
+    Maps an observable defined relative to the isa circuit (where the qubits are indexed based on the "physical" layout of qubits in the device),
+    to the canonical qubit indexing (which is defined by the order of the qubits in the final measurement box of the used circuit)
```

Comment:
I'm still confused by this. Is this an equivalent definition?

```suggestion
    to the canonical qubit indexing (which is defined as the set of qubits measured in the final measurement box of the circuit, sorted according to classical bit index)
```